Gaijinizer patch/Map162.txt

```diff
@@ -318,9 +322,9 @@ would have been so ...#MTLed
 彼女のUSBメモリを調べたときだ。
 > CONTEXT: Map162/events/2/pages/1/74/Dialogue/yuuma
 > CONTEXT: Map162/events/2/pages/2/60/Dialogue/yuuma
-It turned out that Micah was a M.O.W. agent
-It was left after being caught in Camazotz
-It was time to check her USB memory.#MTLed
+I found out Micah was an M.O.W agent after
+investigating a USB stick she left behind when
+she was snatched by Camazotz.#Ron
 > END STRING
```

Comment:
The translation is correct, but think this rewording flows better (feel free to comment on this).
>_**I found out Micah was an M.O.W agent
after checking a USB stick she left behind after
the Camazotz attack.**_